scrolling-tabs.js

```diff
@@ -631,47 +631,58 @@
           smv.decrementMovableContainerLeftPos(newMinPos);
           stc.$movableContainer.stop().animate({ left: smv.getMovableContainerCssLeftVal() }, 'fast');
         }
+
+        var edge;
```

Comment:
Basically the same comment as above: can you add a condition so this block of code (from here down to the call to `handleArrowsClassNames` on line 645) only executes if `scrollingTabsActiveOffset` is set, so other users aren't affected.